data/coalition jobs.txt

```diff
@@ -1154,3 +1154,125 @@ mission "Coalition: From Games 3"
 		payment
 		payment 25000
 		dialog `The passengers thank you for bringing them to <planet> and pay you <payment>.`
+
+
+mission "Coalition: Transport Prisoners 1"
+	job
+	repeat
+	name "Transport Kimek prisoners to <planet>"
+	description "Transport <bunks> Kimek prisoners from <source> for reeducation on <planet>. Payment is <payment>."
+	passengers 20 200 .1
+	to offer
+		has "known to the heliarchs"
+		random < 10
+		"passenger space" > 20
+	source
+		government "Coalition"
+		attributes "kimek"
+	destination "Ring of Power"
```

Comment:
I'm not sure that the `"Ring of Power"` is an appropriate destination - I view it as the Heliarch equivalent of the Pentagon, not a prison ringworld.

Reply:
Ringworlds have more usable area than could ever be used by any of the species described in-universe. There's more than enough space to have military/governmental headquarters and prisons so far apart you'd basically need a spaceship to get between them anyway.

Reply:
I figured it would have to be a Heliarch-controlled area, and Ring of Power makes the most sense to me as a place where more "forceful" applications of Heliarch will would happen.